### PR TITLE
Feat(web): Introduce UNSTABLE_Section component

### PR DESCRIPTION
**Pull request by literat**
## Description

Just using our `docs-Section` as a draft for a new Section component.

All the modifications will be discovered in https://jira.almacareer.tech/browse/DS-1335

### CODE REVIEW

File: packages/web/src/scss/components/UNSTABLE_Section/index.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<section class="UNSTABLE_Section">
+
+</section>
```
Comment (pavelklibani) [inline packages/web/src/scss/components/UNSTABLE_Section/index.html:3]: Should't be something inside the section?

**Review by pavelklibani — APPROVED** [review]

File: packages/web/src/scss/components/UNSTABLE_Section/index.html (hunk 1 above)
Comment (literat) [reply packages/web/src/scss/components/UNSTABLE_Section/index.html]: Added.

File: packages/web/src/scss/components/UNSTABLE_Section/index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  </div>
+</section>
+<!-- Render with all props -->
+
+<section class="UNSTABLE_Section">
+  <h2 class="docs-Heading">
+    My Title <span class="Tag Tag--warning Tag--medium Tag--subtle">My tag</span>
+  </h2>
+  my content
+</section>
```
Comment (pavelklibani) [inline packages/web/src/scss/components/UNSTABLE_Section/index.html:27]: I think you are missing `{{#> web/layout/plain }}` wrapper for this demo, so there is no styling

**Review by pavelklibani — CHANGES_REQUESTED** [review]

File: packages/web/src/scss/components/UNSTABLE_Section/index.html (hunk 2 above)
Comment (literat) [reply packages/web/src/scss/components/UNSTABLE_Section/index.html:29]: 🤦 Fixed.

File: packages/web/src/scss/components/UNSTABLE_Section/_UNSTABLE_Section.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+.UNSTABLE_Section:not(:last-child) {
+    margin-bottom: 2.5rem;
```
Comment (curdaj) [inline packages/web/src/scss/components/UNSTABLE_Section/_UNSTABLE_Section.scss:2]: Is it deliberately stated in rem?  Couldn't we use tokens?

**Review by curdaj — APPROVED** [review]

File: packages/web/src/scss/components/UNSTABLE_Section/_UNSTABLE_Section.scss (hunk 3 above)
Comment (literat) [reply packages/web/src/scss/components/UNSTABLE_Section/_UNSTABLE_Section.scss:2]: This is just a refactor from `docs-Section` to `UNSTABLE_Section`. So I have no intention to change this now. But there will be another iteration after the discovery of the Section component will be done.

**Review by pavelklibani — APPROVED** [review]

**Pull request merged**